We have to remove the vga fallback because this not work on bay-trail and other
new intel onboard graphics.

diff -Naur grub-2.00.org/grub-core/video/i386/pc/vga.c grub-2.00/grub-core/video/i386/pc/vga.c
--- grub-2.00.org/grub-core/video/i386/pc/vga.c	2012-06-06 23:21:02.000000000 +0200
+++ grub-2.00/grub-core/video/i386/pc/vga.c	2014-09-05 21:41:04.790870375 +0200
@@ -122,7 +122,7 @@
 {
   grub_err_t err;
 
-  if ((width && width != VGA_WIDTH) || (height && height != VGA_HEIGHT))
+//  if ((width && width != VGA_WIDTH) || (height && height != VGA_HEIGHT))
     return grub_error (GRUB_ERR_UNKNOWN_DEVICE, "no matching mode found");
 
   framebuffer.temporary_buffer = grub_malloc (VGA_HEIGHT * VGA_WIDTH);
